<?xml version="1.0"?>
<launch>
    
    <node name="Controller" pkg="zeabus_controller" type="origin_control" output="log"/>
    <node name="origin_thruster_2018" pkg="zeabus_controller" type="2018_thrust_mapper_origin.py" output="log">
    </node>
    
</launch>
